ci-operator/step-registry/openshift/image-based/upgrade/seed/create/presubmit/openshift-image-based-upgrade-seed-create-presubmit-commands.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+export PS4='+ $(date "+%T.%N") \011'
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=ERROR
+  -i "${CLUSTER_PROFILE_DIR}/ssh-privatekey")
+
+PULL_SECRET_FILE="/var/run/pull-secret/.dockerconfigjson"
+PULL_SECRET=$(cat ${PULL_SECRET_FILE})
+BACKUP_SECRET_FILE="/var/run/ibu-backup-secret/.backup-secret"
+BACKUP_SECRET=$(cat ${BACKUP_SECRET_FILE})
+SEED_VM_NAME="seed"
+remote_workdir=$(cat ${SHARED_DIR}/remote_workdir)
+instance_ip=$(cat ${SHARED_DIR}/public_address)
+host=$(cat ${SHARED_DIR}/ssh_user)
+ssh_host_ip="$host@$instance_ip"
+
+seed_kubeconfig=${remote_workdir}/ib-orchestrate-vm/bip-orchestrate-vm/workdir-${SEED_VM_NAME}/auth/kubeconfig
+
+SEED_IMAGE_TAG="unknown"
+case $SEED_IMAGE_TAG_FORMAT in
+  "latest")
+    SEED_IMAGE_TAG="latest"
+    ;;
+  "nightly")
+    SEED_IMAGE_TAG="nightly-${SEED_VERSION}-$(date +%F)"
+    ;;
+  "presubmit")
+    SEED_IMAGE_TAG="pre-${PULL_PULL_SHA}"
+    ;;
+  *)
+    echo "Unknown image tag format specified ${SEED_IMAGE_TAG_FORMAT}"
+    exit 1
+    ;;
+esac
+
+echo "${SEED_VM_NAME}" > "${SHARED_DIR}/seed_vm_name"
+
+echo "Creating seed script..."
```

Comment:
Do we want to print the used SEED_IMAGE_TAG here is as well or is this unnecessary?

Reply:
That's a good idea, it was previously there with -x but this is helpful info to have